.travis.yml

```diff
@@ -2,7 +2,6 @@ language: php
 
 php:
   - 7.0
-  - 7.1
```

Comment:
Why are you deleting 7.1 instead of 7.0 ?

Reply:
whoops.. :P my bad.